book/tuto-04-matrices.md

```diff
@@ -47,6 +47,14 @@ target.draw(&vertex_buffer, &indices, &program, &uniforms,
             &Default::default()).unwrap();
 ```
 
+Note that in glium the matrices are column-major.  If we were to write the above matrix in standard mathematical notation, it would look like this (unfortunately, markdown does not let us format things with LaTeX, so this doesn't look nicely formatted):
```

Comment:
Could you write "in glium and opengl"? Because all of opengl assumes column major matrices.

Reply:
Also the comment in the () isn't needed I think.

Reply:
Done!